### ISSUE TITLE
Create feedback after submitting an action

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#297]
Today, submitting an action (changing status or section) doesn't give strong feedback that it went through. This could cause someone to not hit submit or be unsure if it worked. We should should create a stronger use feedback (ex: success alert)

**Comment by lalitha-jonnalagadda** [comment]
@Jacklynn Can we close this given that we have feedback on the complaint details page?

**Comment by Jacklynn** [comment]
I don't think we do, the "apply changes" button is always active even when a user hasn't made changes yesterday. We saw yesterday a CRM user wasn't sure they actually had to click it, it might help if there was more pre and post feedback
